.all-contributorsrc

```diff
@@ -636,6 +636,16 @@
         "bug",
         "content"
       ]
+    },
+    {
+      "login": "tmheunis",
+      "name": "tmheunis",
```

Comment:
hey @tmheunis, let me know if we can credit you with your real name here, or if this line should stay your GitHub log-in

Reply:
ping @tmheunis

Reply:
Thanks @adswa. I have updated this and I will add my name to the zenodo file in this PR.